maintenance/aws-janitor/resources/route53.go

```diff
@@ -44,17 +44,17 @@ func zoneIsManaged(z *route53.HostedZone) bool {
 }
 
 var managedNameRegexes = []*regexp.Regexp{
-	// e.g. api.e2e-61246-dba53.test-cncf-aws.k8s.io.
-	regexp.MustCompile(`^api\.e2e-[0-9]+-`),
+	// e.g. api.e2e-71149fffac-dba53.test-cncf-aws.k8s.io.
+	regexp.MustCompile(`^api\.e2e-[0-9a-z]+-`),
 
-	// e.g. api.internal.e2e-61246-dba53.test-cncf-aws.k8s.io.
-	regexp.MustCompile(`^api\.internal\.e2e-[0-9]+-`),
+	// e.g. api.internal.e2e-71149fffac-dba53.test-cncf-aws.k8s.io.
+	regexp.MustCompile(`^api\.internal\.e2e-[0-9a-z]+-`),
 
-	// e.g. etcd-b.internal.e2e-61246-dba53.test-cncf-aws.k8s.io.
-	regexp.MustCompile(`^etcd-[a-z]\.internal\.e2e-[0-9]+-`),
+	// e.g. etcd-b.internal.e2e-71149fffac-dba53.test-cncf-aws.k8s.io.
+	regexp.MustCompile(`^etcd-[a-z]\.internal\.e2e-[0-9a-z]+-`),
 
-	// e.g. etcd-events-b.internal.e2e-61246-dba53.test-cncf-aws.k8s.io.
-	regexp.MustCompile(`^etcd-events-[a-z]\.internal\.e2e-[0-9]+-`),
+	// e.g. etcd-events-b.internal.e2e-71149fffac-dba53.test-cncf-aws.k8s.io.
+	regexp.MustCompile(`^etcd-events-[a-z]\.internal\.e2e-[0-9a-z]+-`),
```

Comment:
It looks like either the test needs to be updated or the regex needs to be updated to pass the test.

Reply:
Agreed. I just updated the regex to more closely match cluster names from the periodic jobs while excluding the PR-blocking ones, essentially mimic'ing the [logic here](//github.com/kubernetes/test-infra/blob/9ade0873e2bb981f2cc0cf8db656a2180251cf08/scenarios/kubernetes_e2e.py#L282-L285).